### PR TITLE
Added Elmah.Io.Blazor to community doc

### PR DESCRIPTION
**Pull request by ThomasArdal**

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Below is a list of projects and resources from the Blazor community (links in ea
 * [Blazor-State](https://timewarpengineering.github.io/blazor-state/) &ndash; Blazor-State is a client side pipeline architecture utilizing MediatR with plugin behaviors.
 * [Canvas](https://github.com/BlazorExtensions/Canvas) &ndash; HTML5 Canvas API implementation for Microsoft Blazor.
 * [ChartJs.Blazor](https://github.com/mariusmuntean/ChartJs.Blazor) &ndash; Blazor Component that wraps ChartJS.
+* [Elmah.Io.Blazor](https://github.com/elmahio/Elmah.Io.Blazor) &ndash; Logs from Blazor to elmah.io using Microsoft.Extensions.Logging.
```
Comment (danroth27) [inline community.md]: Consider making "elmah.io" a link for folks that aren't familiar with it

**Comment by ThomasArdal** [comment]
Good points. elmah.io now as a link and I've extended the readme with a description of which NuGet packages to install.

**Comment by danroth27** [comment]
@ThomasArdal One minor note: you updated the readme with instructions to install both Microsoft.Extensions.Logging and Elmah.Io.Blazor, but since Elmah.Io.Blazor depends on Microsoft.Extensions.Logging it should be sufficient to just install Elmah.Io.Blazor. All of it's dependencies should get pulled in transitively. But this PR looks fine and can be merged as is.

**Comment by ThomasArdal** [comment]
@danroth27 Ahh, you're right. Apparently, 10 days of Christmas madness, is enough for my brain to reset 🤣 Fixed!

**Pull request merged**